certification/internal/shell/fake_podman_engine_test.go

```diff
@@ -0,0 +1,55 @@
+package shell
```

Comment:
Nit: To @samira-barouti's request elsewhere, I think _this_ can live in its own package. I know you added the test _test suffix to make sure it's only called into scope when testing, but I wonder if we can organize it elsewhere. Up to you if you think it's worth doing now or subsequently.

I also think `"github.com/redhat-openshift-ecosystem/openshift-preflight/cli"` should move elsewhere as well. We can discuss later outside of this PR.

Reply:
Sounds good. Let's go with future us.